### ISSUE TITLE
[FEAT]: WandB 연결

### ISSUE THREAD

**Issue by jerry-ryu** [jerry-ryu/Dacon_Wallpaper_Defect_Classification#1]
# Overview
 - 실험 정리 및 시각화를 위해 WandB 연결
# Todo
### Metrics and Parameters logging
> - [x] WandB 연결
> - [x] Train/Val loss
> - [x] Train/Val ACC. 
> - [x] Train/Val  Weighted F1 Score 
> - [x] class별 평가지표
> - [x] Learning rate
> - [x] Epoch

### Visualization
> - [x] Metric visualiazation logging

# Additional Context
- name convention : 날짜_backbone_실험특이사항 (ex. 0427_efficientnetb0_crossentropyloss)
- WandB 연결하면서 자동으로 logging, naming,  tagging 용이하게 나눠놓기
- WandB 용량 고려해서 Visualization은 local에 주로 저장하기

**Comment by jerry-ryu** [comment]
![image](https://user-images.githubusercontent.com/62556539/236409396-11ecffec-39b4-4aa1-9c43-b4928379bcb6.png)
